### PR TITLE
Lwgeom

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0

### CODE REVIEW

**Comment by agcopenhaver** [comment]
@conda-forge/staged-recipes hate to ping ya, but can I get a review on this one? It's an upstream dependency for another package I'm hoping to bring in

**Comment by ocefpaf** [comment]
> @conda-forge/staged-recipes hate to ping ya

You have to ping to let the team know the recipe is ready for review. This repository is not "monitored."

File: recipes/lwgeom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-rcpp
+    - r-sf >=0.6_0
+    - r-units
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
```
Comment (ocefpaf) [inline recipes/lwgeom/meta.yaml]: Please remove this comment.

File: recipes/lwgeom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  summary: Access to selected functions found in 'liblwgeom' <https://github.com/postgis/postgis/tree/svn-trunk/liblwgeom>,
+    the light-weight geometry library used by 'PostGIS' <http://postgis.net/>.
+  license_family: GPL2
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+  description: Access to selected functions found in liblwgeom, the light-weight geometry library used by PostGIS
+  doc_url: https://cran.r-project.org/web/packages/lwgeom/lwgeom.pdf
+  dev_url: https://github.com/r-spatial/lwgeom/
+extra:
+  recipe-maintainers:
+    - agcopenhaver
```
Comment (ocefpaf) [inline recipes/lwgeom/meta.yaml]: Please add `conda-forge/r` here.

**Pull request merged**